includes/widgets/video.php

```diff
@@ -873,7 +887,14 @@ protected function render() {
 						] );
 					}
 				} else {
-					$this->add_render_attribute( 'image-overlay', 'style', 'background-image: url(' . Group_Control_Image_Size::get_attachment_image_src( $settings['image_overlay']['id'], 'image_overlay', $settings ) . ');' );
+					// When there is an image URL but no ID, it means the overlay image is the placeholder. In this case, get the placeholder URL.
```

Comment:
Not necessarily, in some cases, the URL is an external URL.

Reply:
In which case?